File: resources/permissions.py
'''
	You must replace c11284a125442l587539t1w038651968080-s3bucket-z3wld1oevt6b with your bucket name
'''
import boto3
import json

S3API = boto3.client("s3", region_name="us-west-2") 
bucket_name = "c11284a125442l587539t1w038651968080-s3bucket-z3wld1oevt6b"

policy_file = open("/home/ec2-user/environment/resources/public_policy.json", "r")


S3API.put_bucket_policy(
    Bucket = bucket_name,
    Policy = policy_file.read()
)
print ("DONE")

File: node_10.18.1/permissions.js
(async () => {
	
	var 
		FS = require("fs"),
		AWS_SDK = require("aws-sdk"),
		S3_API = new AWS_SDK.S3({
			apiVersion: "2006-03-01",
			region: "us-west-2"
		}),
		BUCKET_NAME_STR = "<FMI>", //SWAP THIS OUT FOE YOUR BUCKET NAME
		// e.g rh-2020-02-25-s3site
		POLICY_FILE_PATH_STR = "/home/ec2-user/environment/website_security_policy.json",
		params = {},
		policy_str = "",
		response = {};

	params.Bucket = BUCKET_NAME_STR;

	FS.readFile(POLICY_FILE_PATH_STR, async (err, buffer) => {
		if(err){
			return console.log("Error dealing with the policy file", err);
		}
		policy_str = buffer.toString();
		try{
			params.Policy = policy_str;
			response = await S3_API.putBucketPolicy(params).promise();
			if(response){ //will be {} is ok even for repeat calls
				console.log("Done");
			}else{
				console.log("Problem Occurred"); //response === null
			}
		}catch(e){
			console.error("Problem", e);
		}
	});
})();